<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pickCoin</class>
 <widget class="QFrame" name="pickCoin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#pickCoin
{
border-image: url(:/images/rule_background.png);
border-radius:55px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QToolButton" name="coin1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:55px;
background-color: rgba(255, 255, 254, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="coin2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:55px;background-color: rgba(255, 255, 254, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="coin3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:55px;
background-color: rgba(255, 255, 254, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="OK">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;background-color: rgba(255, 255, 254, 50);font: 16pt &quot;Forte&quot;;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
